web/src/message_edit.js

```diff
@@ -1457,3 +1457,9 @@ export function clear_preview_area($element) {
     $row.find(".preview_content").empty();
     $row.find(".markdown_preview").show();
 }
+
+export function focus_into_edit_textarea($element) {
+    const $row = rows.get_closest_row($element);
+    const $msg_edit_content = $row.find(".message_edit_content");
+    $msg_edit_content.trigger("focus");
+}
```

Comment:
Doesn't `last_focused_compose_type_input` already have the right element? 

This seems the same as the caller doing `$last_focused_compose_type_input.trigger("focus");`